src/main/java/software/amazon/documentdb/jdbc/DocumentDbConnection.java

```diff
@@ -143,6 +143,19 @@ public static SshPortForwardingSession createSshTunnel(
         }
     }
 
+    /**
+     * Gets the ssh local port.
+     *
+     * @return the ssh port if it exists; 0 otherwise.
```

Comment:
is 0 still a valid result?

Reply:
nope, should I return something else?

Reply:
Valid port numbers are positive, only.

Reply:
Can we use Optional<Integer>? @birschick-bq , will it work from C++ code?

Reply:
Port 0 (zero) is "reserved" to indicate to allocating API to return a system-assigned (positive) port number. 
https://en.wikipedia.org/wiki/List_of_TCP_and_UDP_port_numbers

If you're more comfortable with a negative number, I'm OK with returning `-1` instead of `0`.

I think using Optional<Integer> might be overkill - but that would also be possible and available from C++.

Reply:
I think returning -1 would work. I like the idea of using Optional and I'm willing to give it a try. However, I'm not sure if Optional can be done in this sprint.

Reply:
update from Slack: if using Optional, returning -1 is not needed

Reply:
@alinaliBQ We need to add a new public method `isSshTunnelActive`.
* Returns true if the `session` class variable is non-null.

Reply:
I have implemented `isSshTunnelActive`, and getSshLocalPort now returns an `Optional `object. Please take a look and let me know if I did it right or not. Thanks!